gallery/gallery/tool/l10n_cli/README.md

```diff
@@ -0,0 +1,4 @@
+# l10n
+
+A command-line application that converts .arb files to .xml files for
+translation consumption.
```

Comment:
Missing  from previous README "Created from templates made available by Stagehand under a BSD-style	
[license](https://github.com/dart-lang/stagehand/blob/master/LICENSE)."

Reply:
That was only there because it was a package 😄 now it’s now it’s separate package anymore so it’s not using any of those generated files.

Reply:
Sounds good